src/resolve/connectionResolver.ts

```diff
@@ -126,8 +149,15 @@ export class ConnectionResolver {
               lastModifiedDate: '',
             }
           );
-        } catch (error) {
-          this.logger.debug((error as Error).message);
+        } catch (err) {
+          // error.message here will be overwritten by 'ts-retry-promise'
+          // Example error.message from the library: "All retries failed" or "Met not retryable error"
+          // 'ts-retry-promise' exposes the actual error on `error.lastError`
+          const error = err as RetryError;
+
+          if (error.lastError && error.lastError.message) {
+            this.logger.debug(error.lastError.message);
+          }
```

Comment:
Should we add a few unit tests to cover the conditions added here? 
- retries and succeeds 
- Hits the retry limit (seemed a bit like exercising the retry lib, but would be good to have a test to verify what happens when it hits the limit). Could just stub the failure condition early verse having to actually call until the lib fails. 
- Not Retryable error thrown

It would make the unit testing a bit cleaner if you refactored the `standardValueSetRecord` specific handling out into a method

Reply:
Added some tests @gbockus-sf 👍